packages/plugins/src/plugins/eigenlayer/jobs/yield.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  EvmNetworkIdType,
+  NetworkId,
+  NetworkIdType,
+} from '@sonarwatch/portfolio-core';
+import { getEvmClient } from '../../../utils/clients';
+import { getEigenLayerOperators } from '../helper';
+import { getAddress } from 'viem';
+import { abi } from '../abi';
+
+export const getYieldPositions = async (networkId: EvmNetworkIdType) => {
+  const client = getEvmClient(networkId);
+
+  const operators = await getEigenLayerOperators();
+  // Get all the strategies addresses from the operators
+  const strategies = Array.from(
+    new Set<`0x${string}`>(
```

Comment:
I believe this should work ?

```suggestion
    new Set<Address>(
```